### DISCUSSION TITLE
Add PR for i18n contribution

### DISCUSSION THREAD

**Discussion by dev-inside** [CorentinTh/enclosed#191] [General]
Hey,
I forked the repository last night to contribute my German translation to the project. Well it's done and I would love to forward it to you, but there was no proper branch where I could add it via pull request. Could you add an branch for it? 

Cheers

**Comment by CorentinTh** [comment]
Hi @dev-inside 
Thank you for contributing 🙏
You normally should be able to create a branch on your fork to propose a pr, I have nothing to do on my side 

More details here : https://docs.github.com/en/get-started/exploring-projects-on-github/contributing-to-a-project#creating-a-branch-to-work-on
